### PR TITLE
[UXIT-1418] Include GitHub PR Template

### PR DESCRIPTION
**Pull request by mirhamasala**
## 📝 Description

Introduces a GitHub pull request template to standardize submissions and reviews.

- **Type:** Documentation

## 🛠️ Key Changes

- Structured template with sections for description, changes, to-do tasks, testing, and screenshots.

## 📎 Additional Notes
[GitHub Docs - Creating a pull request template for your repository](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository)

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+## 📝 Description
```
Comment (mirhamasala) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: @CharlyMartin - Yes or no to emojis? 😊

Comment (CharlyMartin) [reply .github/PULL_REQUEST_TEMPLATE.md]: Let's have emojis! 🧀

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+## 📝 Description
+
+Please include a summary of the changes and the related issue number(s). Provide context and motivation for the change, and describe what problem it solves.
+
+- **Issue:** [Link](#)
```
Comment (CharlyMartin) [inline .github/pull_request_template.md:5]: Is `Link` for the Notion ticket? If so, the Notion / GitHub integration will add it automatically, no?

**Review by CharlyMartin — APPROVED** [review]
Cool, thanks for creating it :)

Looks good to me!

Let's start using this template V1 and see if we need to add/remove sections.

File: .github/pull_request_template.md (hunk 2 above)
Comment (CharlyMartin) [reply .github/pull_request_template.md:5]: Ah I see, I didn't understand because I never use issues, even though we have a few open. Up to you :)

**Pull request merged**